### PR TITLE
Move site to commonwl.org

### LINKED ISSUE #1: deploy to /conferences/2024 ?
Configure a GitHub action to render HTML and publish to https://github.com/common-workflow-language/common-workflow-language.github.io under a `conferences/2024` subfolder using the https://github.com/cwl-bot

### PR DESCRIPTION
**Pull request by alexiswl**
Resolves #1 

Differs from suggestion in https://github.com/common-workflow-language/cwlcon2024/issues/1#issuecomment-1772235987 by adding 'target-directory' parameter.

### CODE REVIEW

**Review by inutano — APPROVED** [review]

File: .github/workflows/deploy_to_commonwl_org.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      - uses: actions/checkout@v4
+
+      # Push website to common-workflow-language.github.io/conferences/2024
+      - name: Pushes to another repository
+        id: push_directory
+        uses: cpina/github-action-push-to-another-repository@v1
+        env:
+          SSH_DEPLOY_KEY: ${{ secrets.SSH_DEPLOY_KEY }}
+        with:
+          source-directory: common-workflow-language.github.io/
```
Comment (mr-c) [inline .github/workflows/deploy_to_commonwl_org.yml:22]: nothing has been written to this directory in this action :-)

Comment (alexiswl) [reply .github/workflows/deploy_to_commonwl_org.yml:22]: Ah interesting, maybe the source-directory needs to be updated to './'?

**Comment by alexiswl** [comment]
https://github.com/common-workflow-language/cwlcon2024/actions/runs/6822736914/job/18555321620

Some notes:

```
Warning: Unexpected input(s) 'commit-message', 'target-branch', 'target-directory', valid inputs are ['entryPoint', 'args', 'source-directory', 'destination-github-username', 'destination-repository-name', 'user-email']
```

```
cp: can't stat 'common-workflow-language.github.io//*': No such file or directory
[master 3c24610] Update from https://github.com/common-workflow-language/cwlcon2024/commit/d15baec57c88ce3ff38baccda7a737f48b5473c0)
 383 files changed, 98471 deletions(-)
 delete mode 100644 404.html
 ....
 delete mode 100644 v1.2/salad.ttl
```

```
fatal: could not read Password for 'https://github.com/': No such device or address
```

**Comment by alexiswl** [comment]
> Warning: Unexpected input(s) 'commit-message', 'target-branch', 'target-directory', valid inputs are ['entryPoint', 'args', 'source-directory', 'destination-github-username', 'destination-repository-name', 'user-email']

I think this is an action version issue, can use `@main` instead but would prefer to fix to at least a major version (but I don't think that's working). So will do v1.7.2 instead as the action version instead.

**Comment by alexiswl** [comment]
We might need to place this work in a subdirectory - https://cpina.github.io/push-to-another-repository-docs/faq.html#how-can-i-copy-the-whole-repository, since copying the whole repository will also copy over this GitHub Action workflow.

**Pull request merged**